src/main/java/org/sigmah/client/ui/presenter/orgunit/OrgUnitCalendarPresenter.java

```diff
@@ -113,10 +112,19 @@ public void onBind() {
 	 * {@inheritDoc}
 	 */
 	@Override
-	public void onPageRequest(final PageRequest request) {
-		final Map<Integer, Integer> calendars = new HashMap<Integer, Integer>();
-		calendars.put(1, getOrgUnit().getId());
-		calendars.put(2, getOrgUnit().getCalendarId());
+	   public void onPageRequest(final PageRequest request) {
+        final Map<Integer, Integer> calendars = new HashMap<Integer, Integer>();
```

Comment:
Yes, that is the expected behavior as expressed in issue #522 : http://www.sigmah.org/issues/view.php?id=522